base/src/main/java/uk/ac/ebi/atlas/model/experiment/baseline/BaselineExpressionPerReplicateProfile.java

```diff
@@ -0,0 +1,20 @@
+package uk.ac.ebi.atlas.model.experiment.baseline;
+
+import uk.ac.ebi.atlas.model.AssayGroup;
+import uk.ac.ebi.atlas.model.Profile;
+
+public class BaselineExpressionPerReplicateProfile extends Profile<AssayGroup, BaselineExpressionPerBiologicalReplicate, BaselineExpressionPerReplicateProfile> {
+
+
+    private BaselineExpressionPerReplicateProfile() {
```

Comment:
Is this zero-args constructor intended for Kryo serialization, right?

Reply:
Yes, it may or may not be actually necessary, I might have just added it because it might be needed for serialization